src/pubkey.h

```diff
@@ -86,9 +86,9 @@ class CPubKey
     }
 
     //! Construct a public key from a byte vector.
-    CPubKey(const std::vector<unsigned char>& vch)
+    CPubKey(const std::vector<unsigned char>& _vch)
     {
-        Set(vch.begin(), vch.end());
+        Set(_vch.begin(), _vch.end());
```

Comment:
Technically this isn't a bug because the local scope should take precedence over class scope.

However clearly it is potentially confusing to rely on scope resolution order for this.

My preference however would be to standardize on using names of the form "argname" + In for this type of situation rather than the more ambiguous _argname.